tests/test_inspiralized.py

```diff
@@ -51,6 +53,6 @@ def test_instructions(self):
             self.harvester_class.instructions(),
         )
 
+    @unittest.expectedFailure
     def test_multiple_instructions(self):
-        # override: this test case legitimately does only contain a single instruction in the source HTML
```

Comment:
I'm going to restore this comment in each affected test case, because there's a small-but-non-zero chance that this pattern could be replicated for other test methods later, and being unambiguous seems worthwhile.